plugin/output/s3/s3.go

```diff
@@ -491,8 +499,10 @@ func (p *Plugin) uploadWork() {
 	for compressed := range p.uploadCh {
 		sleepTime := attemptInterval
 		for {
+			p.logger.Infof("starting upload s3 object: %s to bucket: %s, time: %s", compressed.fileName, compressed.bucketName, time.Now().Format(time.RFC3339))
```

Comment:
You can change log format and time.now will be useless

Reply:
Added to all logs.